ConsentViewController/Classes/ConsentViewController.swift

```diff
@@ -492,11 +501,32 @@ import Reachability
         consentDelegate?.onConsentReady(controller: self)
     }
 
-    private func onErrorOccurred(error: ConsentViewControllerError) {
+    internal func onErrorOccurred(error: ConsentViewControllerError) {
         releaseWebViewHandlers()
+        rollBarAnalytics(error: error.description)
         consentDelegate?.onErrorOccurred(error: error)
     }
-
+    
+     /// This method is used to send the details to rollbar analytics about the error message with other deatils.
+    internal func rollBarAnalytics(error: String) {
+        if !isDisableRollbar {
+        let configuration = RollbarConfiguration()
+        configuration.crashLevel = "critical"
+        configuration.environment = "production"
+        Rollbar.initWithAccessToken("8c9341f5b0cd4701b43fd06237b0b660", configuration: configuration)
+        Rollbar.critical(withMessage: error, data: [
```

Comment:
By adding `Rollbar` directly in our code we're strongly coupling our code with it. 
We should implement a protocol and a wrapper around Rollbar. This is will make it easier to test and decouple our code from `Rollbar`.